app/views/contexts/calculated_summary_context.py

```diff
@@ -126,8 +124,8 @@ def _build_calculated_summary_section(
         return {"id": section_id, "groups": [{"id": group["id"], "blocks": blocks}]}
 
     def _remove_unwanted_questions_answers(
-        self, block, answer_ids_to_keep, current_location
-    ) -> str:
+        self, block: Dict, answer_ids_to_keep: list, current_location: Location
```

Comment:
Try to use existing classes as types, so `block: Block`

Reply:
👍